modules/ppcp-wc-gateway/src/Gateway/OXXO/OXXOGateway.php

```diff
@@ -0,0 +1,199 @@
+<?php
+/**
+ * The OXXO Gateway
+ *
+ * @package WooCommerce\PayPalCommerce\WcGateway\Gateway
+ */
+
+declare(strict_types=1);
+
+namespace WooCommerce\PayPalCommerce\WcGateway\Gateway\OXXO;
+
+use Psr\Log\LoggerInterface;
+use WC_Payment_Gateway;
+use WooCommerce\PayPalCommerce\ApiClient\Endpoint\OrderEndpoint;
+use WooCommerce\PayPalCommerce\ApiClient\Exception\PayPalApiException;
+use WooCommerce\PayPalCommerce\ApiClient\Exception\RuntimeException;
+use WooCommerce\PayPalCommerce\ApiClient\Factory\PurchaseUnitFactory;
+use WooCommerce\PayPalCommerce\ApiClient\Factory\ShippingPreferenceFactory;
+
+/**
+ * Class OXXOGateway.
+ */
+class OXXOGateway extends WC_Payment_Gateway {
+	const ID = 'ppcp-oxxo-gateway';
+
+	/**
+	 * The order endpoint.
+	 *
+	 * @var OrderEndpoint
+	 */
+	protected $order_endpoint;
+
+	/**
+	 * The purchase unit factory.
+	 *
+	 * @var PurchaseUnitFactory
+	 */
+	protected $purchase_unit_factory;
+
+	/**
+	 * The shipping preference factory.
+	 *
+	 * @var ShippingPreferenceFactory
+	 */
+	protected $shipping_preference_factory;
+
+	/**
+	 * The logger.
+	 *
+	 * @var LoggerInterface
+	 */
+	protected $logger;
+
+	/**
+	 * OXXOGateway constructor.
+	 *
+	 * @param OrderEndpoint             $order_endpoint The order endpoint.
+	 * @param PurchaseUnitFactory       $purchase_unit_factory The purchase unit factory.
+	 * @param ShippingPreferenceFactory $shipping_preference_factory The shipping preference factory.
+	 * @param LoggerInterface           $logger The logger.
+	 */
+	public function __construct(
+		OrderEndpoint $order_endpoint,
+		PurchaseUnitFactory $purchase_unit_factory,
+		ShippingPreferenceFactory $shipping_preference_factory,
+		LoggerInterface $logger
+	) {
+		$this->id = self::ID;
+
+		$this->method_title       = __( 'OXXO', 'woocommerce-paypal-payments' );
+		$this->method_description = __( 'OXXO is a Mexican chain of convenience stores.', 'woocommerce-paypal-payments' );
+
+		$gateway_settings  = get_option( 'woocommerce_ppcp-oxxo-gateway_settings' );
+		$this->title       = $gateway_settings['title'] ?? $this->method_title;
+		$this->description = $gateway_settings['description'] ?? __( 'OXXO allows you to pay bills and online purchases in-store with cash.', 'woocommerce-paypal-payments' );
```

Comment:
You can use `$this->get_option`

https://github.com/woocommerce/woocommerce-paypal-payments/blob/ec8ba1509276615ea0ca1fed8641189d7a8ca876/modules/ppcp-wc-gateway/src/Gateway/CardButtonGateway.php#L180-L181